inc/authldap.class.php

```diff
@@ -1600,14 +1745,21 @@ static function getAllUsers(array $options, &$results, &$limitexceeded) {
          $values['order'] = "ASC";
       }
       $ds = $config_ldap->connect();
+      $field_for_sync = $config_ldap->getLdapIdentifierToUse();
```

Comment:
i count 3 $config_ldap->getDatabaseIdentifierToUse() in the code below.
Maybe like $field_for_sync, we could use a var to improve readability ($field_for_db ?)